lib/benefits.js

```diff
@@ -9,104 +9,82 @@ export function getBenefitData(id) {
   }
 
   return benefit;
-
-  // if (id == "oas") {
-  //   return {
-  //     name: "Old age security",
-  //     type: "oas type",
-  //     outcomes: "oas outcomes",
-  //     provider: "oas provider",
-  //   };
-  // }
-  // if (id == "ei") {
-  //   return {
-  //     name: "Employment insurance",
-  //     type: "ei type",
-  //     outcomes: "ei outcomes",
-  //     provider: "ei provider",
-  //   };
-  // }
-  // return null;
 }
 
 //TODO: get benefit ids data from better source
 export function getBenefitIds() {
-  return getBenefits().map((benefit) => {
-    return {
-      params: {
-        id: benefit.id,
-      },
-    };
-  });
+  return [
+    "benefit-1",
+    "benefit-2",
+    "benefit-3",
+    "benefit-4",
+    "benefit-5",
+    "benefit-6",
+  ];
+  // return getBenefits(t).map((benefit) => {
+  //   return {
+  //     params: {
+  //       id: benefit.id,
+  //     },
+  //   };
+  // });
```

Comment:
I left this in since I wasn't sure how Strapi works but this concept is more realistic to what it should be. I currently just have it hard coded because of language issues but it needs to be updated this sprint